src/backend/assets/js/session.js

```diff
@@ -0,0 +1,66 @@
+$(document).ready(function() {
+  var sideslider = $('[data-toggle=collapse-side]');
+  var sel = sideslider.attr('data-target');
+  var top = $('header[role=banner]').outerHeight();
+  var downButton = $('#down-button');
+  var sessionElem = $('.single-session');
+  var sessionId = Number(sessionElem.attr('id'));
+  var maxNoOfSessions = 1000;
```

Comment:
Thousand seemed like a reasonable number so I set to it :grinning: . If it becomes more than that, then there shouldn't be any serious problem since the arrays are dynamic in nature in JS. We have also used the same number for tracks, rooms and schedule pages also. If we want to change it, we can perhaps make an issue and then fix it on all the pages at once. Please suggest. Thanks!!